src/harness/harness.c

```diff
@@ -181,10 +181,12 @@ void Harness_Init(int* argc, char* argv[]) {
     } else {
         root_dir = OS_Dirname(argv[0]);
     }
-    printf("Using root directory: %s\n", root_dir);
-    result = chdir(root_dir);
-    if (result != 0) {
-        LOG_PANIC("Failed to chdir. Error is %s", strerror(errno));
+    if (root_dir != NULL && strlen(root_dir) > 1) {
```

Comment:
On discord, you said `"."` worked.
What about only ignoring an empty `root_dir`?
```suggestion
    if (root_dir != NULL && root_dir[0] != '\0') {
```

Reply:
My thinking was if the value is `.`, we don't need to call chdir since it is a no-op. I suppose I could do a `strcmp(root_dir, '.')` to make that more obvious

Reply:
It's a no-op, but I don't think we don't care. `chdir`'s implementation should be able to handle it.
The current change would disallow one-letter directories.
An unlikely directory structure, but possible.